pkg/crypto/aes.go

```diff
@@ -172,28 +158,53 @@ func decodeHex(passphrase string, ciphertext string) ([]byte, []byte, []byte, er
 	return salt, iv, data, nil
 }
 
-// Decrypt ciphertext with a passphrase
-func Decrypt(passphrase string, ciphertext []byte, encoding string) (string, error) {
+// Decrypt ciphertext with a passphrase.
+// Formats:
+// 1) `encoding:text`
+// 2) `text`
+func Decrypt(passphrase string, ciphertext []byte) (string, error) {
 	var salt []byte
 	var iv []byte
 	var data []byte
-	if encoding == "base64" {
+
+	cParts := strings.SplitN(string(ciphertext), ":", 3)
+	rawEncoding := ""
+	var ciphertextData string
```

Comment:
supernit: Any reason why you initialized these vars differently? They will all be initialized as empty strings as you have them now

Reply:
None whatsoever. Will update this.

Reply:
Cool 👍 Just wanted to make sure there wasn't a misunderstanding here